thunder/core/prims.py

```diff
@@ -254,7 +254,8 @@ class PrimIDs(Enum):
     EMBEDDING_BACKWARD = auto()
     LINEAR = auto()
     PAD = auto()
-    BATCH_NORM = auto()
+    ADAPTIVE_AVG_POOL2D = auto()
+    ADAPTIVE_AVG_POOL2D_BACKWARD = auto()
```

Comment:
Some of the prims are large and complicated operations. I think we started to put things like `prims.embedding` and `prims.embedding_backward` because we didn't have a good understanding of how to go from a PyTorch op to something executable without decomposing it. Now we do have a direct path from a PyTorch op to its execution, see `is_prim=True` usage in `thunder/torch/__init__.py`.

I think it's better to define these straightforward meta functions in the `thunder/torch/__init__.py` file directly, leaving `prims.py` unmodified.